compiler_test.go

```diff
@@ -1967,7 +1967,7 @@ func TestEmptyCodeError(t *testing.T) {
 	if _, err := New().RunString(`i`); err == nil {
 		t.Fatal("Expected an error")
 	} else {
-		if e := err.Error(); e != "ReferenceError: 'i' is not defined at <eval>:1:1(0)" {
+		if e := err.Error(); e != "ReferenceError: 'i' is not defined" {
```

Comment:
Not including the position from an exception's error message is more in line with node and otto.

Reply:
👍🏻